package.json

```diff
@@ -11,7 +11,7 @@
     "build": "yarn workspaces foreach -p --topological-dev --exclude '{root-workspace}' run build",
     "build:docs": "typedoc --theme default --out docs/api --tsconfig tsconfig.docs.json && touch docs/.nojekyll",
     "bundle": "yarn workspace @kiltprotocol/sdk-js run bundle",
-    "clean": "yarn workspaces foreach -p --exclude '{root-workspace}' run clean",
+    "clean": "rm -rf tests/dist && yarn workspaces foreach -p --exclude '{root-workspace}' run clean",
```

Comment:
we're using `rimraf` elsewhere, I think that is to be more platform-agnostic

Reply:
yeah I'd say so. I don't know the exact reasons why we're working with `rimraf`, but whatever it solves, it should solve it here too.

Reply:
Done!